CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_marketo v0.8.0
```

Comment:
Be sure to update the CHANGELOG with any relevant changes from this (or the source) PR review.

Reply:
Done.